libs/backend/src/cast_vote_records/export.ts

```diff
@@ -679,12 +698,36 @@ export async function exportCastVoteRecordsToUsbDrive(
     scannerStore.clearCastVoteRecordHashes();
   }
 
-  const exportDirectoryPathRelativeToUsbMountPoint =
-    await getExportDirectoryPathRelativeToUsbMountPoint(exportContext);
+  const isRecoveryExport =
+    exportOptions.scannerType === 'precinct' && exportOptions.isRecoveryExport;
```

Comment:
What are we confirming by checking the `scannerType` is `precinct`?

Reply:
Yup exactly! VxCentralScan doesn't use continuous export, so the notion of a recovery export doesn't make sense on it